.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
     "python.testing.pytestEnabled": true,
-    "python.formatting.provider": "black"
-}
+    "python.formatting.provider": "black",
+    "editor.rulers": [
+        80
+    ],
```

Comment:
oh also remove the trailing comma here, it's not valid json and it looks like it will cause warnings.
https://github.com/microsoft/vscode/issues/102061

Reply:
SG, done.